### PR TITLE
Added support to watch uploads

### PR DESCRIPTION
**Pull request by r-gruber**
Twitch added a possibilty to upload videos a while ago. VODs can now
either be of type highlight, archive or upload. In order to be able
to see them the query must use the new 'broadcast_type' parameter.

This plugin will now display all uploads of a channel in a sepate list
similar to how it was done so far for past broadcasts or highlights.

### CODE REVIEW

**Review by SpaceOgre — CHANGES_REQUESTED** [review]
Hi, 

Thanks for the commit :) 

I think it would make more sense to change broadcast and upload parameters to just use broadcastType for the ChannelVodList straight away. Less mapping needed.

**Comment by r-gruber** [comment]
I replaced the boolean parameters of the ChannelVodList method with a single broadcastType parameter. The used broadcastType will also be used to determine the correct title of the page.

**Review by SpaceOgre — APPROVED** [review]

File: Contents/Code/__init__.py [hunk 1]
```diff
@@ -264,19 +269,9 @@ def FollowedChannelsList(apiurl=None, limit=100, **kwargs):
 
 
 @route(PREFIX + '/channel/vods', broadcasts=bool, uploads=bool, limit=int)
```
Comment (SpaceOgre) [inline Contents/Code/__init__.py]: You forgot to change the route parameters.

**Review by SpaceOgre — CHANGES_REQUESTED** [review]

**Comment by SpaceOgre** [comment]
Except the route parameters it looks great :) 
Will merge when it is fixed.

**Comment by SpaceOgre** [comment]
Thanks for your contribution :)

**Pull request merged**